.github/workflows/jupyterlite.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install jupyterlite[all] libarchive-c build pyodide-build
+          python -m pip install jupyterlite[all]==0.1.0b17 libarchive-c build pyodide-build
```

Comment:
Should we add a new file called requirements-jupyterlite so as to control better this?
What do people think?

Reply:
My opinion on that is that it should not be thought of as a requirement of xDSL.
JupyterLite is only used to generate a static Jupyter distribution, which I don't see making sense locally for example, where one can simply `jupyter lab`
I think this only makes sense in the CI, and assuming that, getting that in another file would just add an indirection and the induced maintenance headache.